woo-razorpay.php

```diff
@@ -2467,6 +2466,98 @@ public function getVersionMetaInfo($data = array())
             }
         }
 
+        public function prepayCODOrder(array $payload): WP_REST_Response
```

Comment:
Can we move the prepay to cod related functions to a separate file? This will help us maintain feature-wise functions.

Reply:
moved it to separate file